app/collections/[slug]/page.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Metadata } from "next";
+import PageLayout from "app/components/pageLayout/pageLayout";
+import Collections from "app/components/collections/collections";
+import { useSearchParams } from "next/navigation";
+
+type CollectionProps = {
+  params: { slug: string };
+};
+
+export async function generateMetadata({
+  params,
+}: CollectionProps): Promise<Metadata> {
+  const slug = params.slug;
+  return {
+    title: `${slug} - NYPL Digital Collections`,
+  };
+}
+
+export default function Lane({ params }: CollectionProps) {
+  return (
+    <PageLayout
+      activePage="lane"
+      breadcrumbs={[
+        { text: "Home", url: "/" },
+        { text: "All Collections", url: "/collections" },
+        {
+          text: `${params.slug}`,
+          url: `/collections/${params.slug}`,
+        },
+      ]}
+    >
+      <Collections slug={params.slug} />
```

Comment:
NItpick: I'd rename this as `CollectionsPage` so it's not confusing if there's a component that is a list of collections called `Collections`. This current component includes the search and other elements on the page so that's why I'm thinking `CollectionsPage` might be better.